src/components/app/details/triggerView/workflow/nodes/triggerCINode.tsx

```diff
@@ -100,7 +102,14 @@ export class TriggerCINode extends Component<TriggerCINodeProps> {
 
     renderCardContent(context) {
         const hideDetails = this.hideDetails(this.props.status?.toLowerCase())
-
+        let _selectedPipeline = this.props.filteredCIPipelines?.find((_ciPipeline) => _ciPipeline?.id == this.props.id)
```

Comment:
Ca we extract these out to some separate function so that it can be reused.

Reply:
These two are used differently at two different places.